frontend/src/components/organisms/Navbar/Navbar.tsx

```diff
@@ -12,40 +17,64 @@ const Navbar = ({ menuItems }: NavbarProps): ReactElement<null, 'div'> | null =>
 
     const [languageSelectorOpen, setLanguageSelectorOpen] = useState<boolean>(false);
 
+    const handleButtonClick = async () => {
+        const environment = createRelayEnvironment({});
+
+        await fetchQuery<sendTemplatedEmailQuery>(environment, SendTemplatedEmailQuery, {
+            emailTo: 'jakub.bednar@sklinet.com',
+            emailTemplate: 1,
+            variables: {
+                email: 'jakub.bednar@sklinet.com',
+                firstName: 'Jakub',
+            },
+        }).toPromise();
+    };
```

Comment:
@jbednar97 toto je zle, jedna vec, ze tieto veci nevolame nikdy z FE, ale volame api routu, ktora to spracovava. Druha vec, ze to tlacitko tu nema absolutne co hladat a vobec si nemal sahat do navigacie, bavili sme sa o tom, ze to bude akcia po vyplneni napriklad kontaktneho formularu a nie, ze do navigacie das tlacitko na odoslanie mailu.

Reply:
Vytvoril som separatnu komponentu (formulár s menom a e-mailom), a napojil to na API routu kde je fetchQuery